src/main/resources/init_db.sql

```diff
@@ -0,0 +1,11 @@
+CREATE DATABASE library;
+USE library;
+CREATE TABLE books (
+    id BIGINT PRIMARY KEY,
+    title VARCHAR(255),
+    price BIGINT(45)
+);
+ALTER TABLE books MODIFY COLUMN

```

Comment:
i`m not sure if this is a required code after 7 line

Reply:
agree